workspaces/src/network/mod.rs

```diff
@@ -10,13 +10,16 @@ mod server;
 mod testnet;
 pub(crate) mod variants;
 
+pub(crate) use sandbox::SandboxPatchAcessKeyBuilder; //not needed directly outside of the crate
+pub(crate) use sandbox::SandboxPatchStateAccountBuilder; //not needed directly outside of the crate
+pub(crate) use sandbox::SandboxPatchStateBuilder; //not needed directly outside of the crate
 pub(crate) use variants::DEV_ACCOUNT_SEED;
 
-pub use self::betanet::Betanet;
-pub use self::info::Info;
-pub use self::mainnet::Mainnet;
-pub use self::sandbox::Sandbox;
-pub use self::testnet::Testnet;
-pub use self::variants::{
+pub use betanet::Betanet;
+pub use info::Info;
+pub use mainnet::Mainnet;
+pub use sandbox::Sandbox;
+pub use testnet::Testnet;
+pub use variants::{
```

Comment:
I'm a bit confused by this suggestion, what would cause this to break? This is just being more explicit about where it's coming from, if anything the latter would be open to being broken?